tests/auth_tests/test_auth_backends.py

```diff
@@ -574,6 +578,20 @@ def test_changed_backend_settings(self):
             self.assertIsNotNone(user)
             self.assertTrue(user.is_anonymous)
 
+    @override_settings(AUTHENTICATION_BACKENDS=[
+        'auth_tests.test_auth_backends.SomeModelBackend',
+        'django.contrib.auth.backends.ModelBackend'])
+    def test_missing_get_user(self):
+        """
+        In force_login() use only authentication backend which has get_user() method.
```

Comment:
force_login() skips authentication backends without a get_user() method.
(wrap docstrings at 79 characters per the style guide)